.github/workflows/graal-ci.yml

```diff
@@ -0,0 +1,28 @@
+---
+name: CI Build in GraalVM with tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: graalvm/setup-graalvm@v1
+        with:
+          java-version: 17
+          cache: maven
+
+      - name: build
+        env:
+          MAVEN_CONFIG: "-Dbasepom.check.skip-enforcer=false -B"
```

Comment:
the ci.yml has `-B -fae`

Reply:
Yes, I left this out intentionally, it is easier to track down errors when they are always at the end of the build log